README.md

```diff
@@ -12,77 +12,73 @@ INSTALLED_APPS = [
 ] + get_core_apps([
     "oscar_elasticsearch.search", # replace standard oscar app
 ]) + [
+    "widget_tweaks",
     "wagtail.core",
-    "wagtail.contrib.search_promotions", # allow search result display and such
     "wagtail.search",
 ]
 ```
 
 Settings required to replace the search app.
 
 ```Python
-OSCAR_ELASTICSEARCH_QUERY_PAGE_SIZE = 100
 OSCAR_PRODUCT_SEARCH_HANDLER = "oscar_elasticsearch.search.search_handlers.ProductSearchHandler"
-OSCAR_SEARCH = {
```

Comment:
Why this setting was removed?

Reply:
The settings are nolonger grouped but separate values, with sane defaults.